### PR TITLE
[ZH] Update `Tournaments`

### PR DESCRIPTION
**Pull request by untitledunrevised**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Walavouchey** [comment]
hi please refrain from updating the branch yourself as it will be done right before merging anyway. it only creates unnecessary merge commits in between actual changes.

image

File: wiki/Tournaments/zh.md [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
-outdated_translation: true
-outdated_since: 70fd166193a632f68f3c3ec84d64c07a8dbfbd38
----
-
 # 锦标赛
 
-本页面展示锦标赛。[论坛链接](https://osu.ppy.sh/community/forums/55)
+本页面展示锦标赛。锦标赛有专门的论坛，可以通过[此链接](https://osu.ppy.sh/community/forums/55)找到。
+
+请访问[官方锦标赛支持页面](https://osu.ppy.sh/wiki/en/Tournaments/Official_support)，详细了解如何为您的锦标赛申请官方支持（徽章、筛选流程等）。
```
Comment (YumeMuzi) [inline wiki/Tournaments/zh.md:5]:
```suggestion
请访问[官方锦标赛支持页面](/wiki/Tournaments/Official_support)，详细了解如何为您的锦标赛申请官方支持（徽章、筛选流程等）。
```

File: wiki/Tournaments/zh.md [hunk 2]
```diff
@@ -230,109 +557,245 @@ osu! 世界杯是国家间的 4v4 组队对抗赛，赛制为单败淘汰制。
...
+| [Chinese osu!mania 4K Tournament 2017](CMT_4K/2017) | 2017-06-24 | 2017-08-06 | ::{ flag=TW }:: [- Choma -](https://osu.ppy.sh/users/4469570) | ::{ flag=CN }:: [\[Crz\]MemoryI](https://osu.ppy.sh/users/8179131) | ::{ flag=CN }:: [\[Crz\]ScSolAr](https://osu.ppy.sh/users/1591215) |
+| [Chinese osu!mania 4K Tournament 2018](CMT_4K/2018) | 2018-09-10 | 2018-11-25 | ::{ flag=MY }:: [cheewee10](https://osu.ppy.sh/users/4477497) | ::{ flag=HK }:: [\[Crz\]Murasame](https://osu.ppy.sh/users/11563203) | ::{ flag=SG }:: [ByeForNow](https://osu.ppy.sh/users/7199159) |
+
+#### GB Cup
 
-### ![](/wiki/shared/mode/mania.png) osu!mania LN Tournament
+| 名称 | 开始时间 | 结束时间 | ![冠军][GCrown] | ![亚军][SCrown] | ![季军][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| [GB Cup 2022 Spring](GBC/GBC_2022_Spring) | 2022-04-09 | 2022-05-15 | 金戈铁马的鲜丰水果摊 | 太美丽了理塘 | GBC最强阵容9个Miyes |
+| [GB Cup 2022 Autumn](GBC/GBC_2022_Autumn) | 2022-10-15 | 2022-12-18 | 无所谓绿皮龙会出手的 | BMS精灵 | 	俄亥俄州立大学躁狂播放器 |
```
Comment (YumeMuzi) [inline wiki/Tournaments/zh.md:652]:
```suggestion
| [GB Cup 2022 Autumn](GBC/GBC_2022_Autumn) | 2022-10-15 | 2022-12-18 | 无所谓绿皮龙会出手的 | BMS精灵 | 俄亥俄州立大学躁狂播放器 |
```

File: wiki/Tournaments/zh.md [hunk 3]
```diff
@@ -1,48 +1,49 @@
...
+| [osu! 世界杯 2014](OWC/2014) | 2014-10-02 | 2014-12-14 | ::{ flag=JP }:: 日本 | ::{ flag=PL }:: 波兰 | ::{ flag=DE }:: 德国 |
+| [osu! 世界杯 2015](OWC/2015) | 2015-10-01 | 2015-12-13 | ::{ flag=US }:: 美国 | ::{ flag=CN }:: 中国 | ::{ flag=PL }:: 波兰 |
+| [osu! 世界杯 2016](OWC/2016) | 2016-10-02 | 2016-12-18 | ::{ flag=US }:: 美国 | ::{ flag=CN }:: 中国 | ::{ flag=KR }:: 韩国 |
+| [osu! 世界杯 2017](OWC/2017) | 2017-10-13 | 2017-12-17 | ::{ flag=PL }:: 波兰 | ::{ flag=US }:: 美国 | ::{ flag=GB }:: 英国 |
+| [osu! 世界杯 2018](OWC/2018) | 2018-11-10 | 2018-12-16 | ::{ flag=US }:: 美国 | ::{ flag=GB }:: 英国 | ::{ flag=DE }:: 德国 |
+| [osu! 世界杯 2019](OWC/2019) | 2019-11-09 | 2019-12-22 | ::{ flag=US }:: 美国 | ::{ flag=KR }:: 韩国 | ::{ flag=GB }:: 英国 |
+| [osu! 世界杯 2020](OWC/2020) | 2020-10-24 | 2020-12-06 | ::{ flag=US }:: 美国 | ::{ flag=DE }:: 德国 | ::{ flag=CA }:: 加拿大 |
+| [osu! 世界杯 2021](OWC/2021) | 2021-10-09 | 2021-11-28 | ::{ flag=US }:: 美国 | ::{ flag=DE }:: 德国 | ::{ flag=KR }:: 韩国 |
+| [osu! 世界杯 2021](OWC/2022) | 2022-10-15 | 2022-11-27 | ::{ flag=US }:: 美国 | ::{ flag=KR }:: 韩国 | ::{ flag=DE }:: 德国 |
```
Comment (YumeMuzi) [inline wiki/Tournaments/zh.md:29]:
```suggestion
| [osu! 世界杯 2022](OWC/2022) | 2022-10-15 | 2022-11-27 | ::{ flag=US }:: 美国 | ::{ flag=KR }:: 韩国 | ::{ flag=DE }:: 德国 |
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/zh.md (hunk 2 above)
Comment (RushFTK) [reply wiki/Tournaments/zh.md]: 这比赛冠军怎么是这三个队。。。

File: wiki/Tournaments/zh.md [hunk 4]
```diff
@@ -98,130 +132,423 @@ osu! 世界杯是国家间的 4v4 组队对抗赛，赛制为单败淘汰制。
...
+| [Yaz's Summer Cup 2018](YSC/2018) | 2018-06-16 | 2018-08-12 | ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050) | ::{ flag=KR }:: [Yaong](https://osu.ppy.sh/users/1883865) | ::{ flag=GB }:: [Doomsday](https://osu.ppy.sh/users/18983) |
+| [Yaz's Summer Cup 2019](YSC/2019) | 2019-07-06 | 2019-08-25 | ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050) | ::{ flag=US }:: [thaibuy](https://osu.ppy.sh/users/4250296) | ::{ flag=US }:: [DigitalHypno](https://osu.ppy.sh/users/4384207) |
+| [Yaz's Summer Cup 2021](YSC/2021) | 2021-06-25 | 2021-08-22 | ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) | ::{ flag=CL }:: [Mathi](https://osu.ppy.sh/users/5339515) | ::{ flag=GB }:: [Bubbleman](https://osu.ppy.sh/users/5182050) |
+
+#### Yuri Cup
+
+| 名称 | 开始时间 | 结束时间 | ![冠军][GCrown] | ![亚军][SCrown] | ![季军][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| [Yuri Cup](YC/1) | 2020-02-22 | 2020-04-05 | Corim's Bad Dream | a la verga | Rtzero x Rtzer0 |
+
```
Comment (YumeMuzi) [inline wiki/Tournaments/zh.md:415]:
```suggestion

#### Yuyuko's Hidden Cup

| 名称 | 开始时间 | 结束时间 | ![冠军][GCrown] | ![亚军][SCrown] | ![季军][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| Yuyuko's Hidden Cup Season 1 | 2022-05-17 | 2022-07-04 | 都怪幽神 | HD怎么打 | 队名叫有新意的名字 / 请在此处填写队名 |
| [Yuyuko's Hidden Cup Season 2](YHC/2) | 2022-11-26 | 2023-02-05 | 懒人三项 | 老年娱乐 | 已经结束啦 |

```

File: wiki/Tournaments/en.md [hunk 5]
```diff
@@ -649,7 +649,7 @@ Unofficial tournaments/competitions hosted by the communities.
 | Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
 | :-- | :-- | :-- | :-- | :-- | :-- |
 | [GB Cup 2022 Spring](GBC/GBC_2022_Spring) | 2022-04-09 | 2022-05-15 | 金戈铁马的鲜丰水果摊 | 太美丽了理塘 | GBC最强阵容9个Miyes |
-| [GB Cup 2022 Autumn](GBC/GBC_2022_Autumn) | 2022-10-15 | 2022-12-18 | *TBD* | *TBD* | *TBD* |
+| [GB Cup 2022 Autumn](GBC/GBC_2022_Autumn) | 2022-10-15 | 2022-12-18 | 猫猫面馆 | 黎曼猜想 | 均衡教派 |
```
Comment (YumeMuzi) [inline wiki/Tournaments/en.md:652]: plz keep de/fr.md in sync or just add an `outdated_wiki` mark following https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#front-matter

**Review by YumeMuzi — CHANGES_REQUESTED** [review]

**Review by YumeMuzi — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**